src/ServiceControl.Audit/Auditing/AuditIngestion.cs

```diff
@@ -78,7 +78,7 @@ FailedAuditImport FailedMessageFactory(FailedTransportMessage msg)
 
         Task OnCriticalError(string failure, Exception exception)
         {
-            logger.Warn($"OnCriticalError. '{failure}'", exception);
+            logger.Fatal($"OnCriticalError. '{failure}'", exception);
```

Comment:
@ramonsmits Doesn't look like we introduced it - it's been there for a while.

Reply:
Depends on your POV I think but a critical errors means all is lost, hence the attempt to restart which still can not have any positive effect and have the instance remain in an unhealthy state

Reply:
Sure - I'm just guessing as to the original reasoning behind the warning vs fatal. I have no issue with it being changed to a fatal, as a fatal error has occurred which caused the instance to stop, and hopefully with any luck it will recover from it.